src/main/java/frc/team2412/robot/commands/launcher/FullTargetCommand.java

```diff
@@ -0,0 +1,93 @@
+package frc.team2412.robot.commands.launcher;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.interpolation.InterpolatingTreeMap;
+import edu.wpi.first.wpilibj.Filesystem;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.team2412.robot.Controls;
+import frc.team2412.robot.Robot;
+import frc.team2412.robot.subsystems.DrivebaseSubsystem;
+import frc.team2412.robot.subsystems.IntakeSubsystem;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+import frc.team2412.robot.util.LauncherDataLoader;
+import frc.team2412.robot.util.LauncherDataPoint;
+import java.nio.file.FileSystems;
+import java.util.function.BooleanSupplier;
+
+public class FullTargetCommand extends Command {
+
+	private static final InterpolatingTreeMap<Double, LauncherDataPoint> LAUNCHER_DATA =
+			LauncherDataLoader.fromCSV(
+					FileSystems.getDefault()
+							.getPath(Filesystem.getDeployDirectory().getPath(), "launcher_data.csv"));
+	private static final float YAW_TARGET_VIBRATION_TOLERANCE = 3; // degrees
```

Comment:
Minor: Why is this a float and not a double?

Reply:
it's a bad habit